src/User/src/ConfigProvider.php

```diff
@@ -48,12 +48,12 @@ public function getDependencies(): array
             'factories'  => [
                 AuthenticationService::class => AuthenticationServiceFactory::class,
                 AuthenticationAdapter::class => AuthenticationAdapterFactory::class,
-                UserController::class        => AnnotatedServiceFactory::class,
-                AccountController::class     => AnnotatedServiceFactory::class,
-                UserService::class           => AnnotatedServiceFactory::class,
-                UserRoleService::class       => AnnotatedServiceFactory::class,
-                UserRepository::class        => AnnotatedRepositoryFactory::class,
-                UserRoleRepository::class    => AnnotatedRepositoryFactory::class,
+                UserController::class        => AttributedServiceFactory::class,
+                AccountController::class     => AttributedServiceFactory::class,
+                UserService::class           => AttributedServiceFactory::class,
+                UserRoleService::class       => AttributedServiceFactory::class,
+                UserRepository::class        => AttributedRepositoryFactory::class,
+                UserRoleRepository::class    => AttributedRepositoryFactory::class,
```

Comment:
Error on login:
> Unable to resolve service "Frontend\User\EventListener\UserAvatarEventListener" to a factory; are you certain you provided it during configuration?

```suggestion
                UserRoleRepository::class         => AttributedRepositoryFactory::class,
                UserAvatarEventListener::class => AttributedServiceFactory::class,
```

Also, don't forget to add `use Frontend\User\EventListener\UserAvatarEventListener;`.